javascript/packages/orchestrator/src/networkNode.ts

```diff
@@ -209,7 +209,7 @@ export class NetworkNode implements NetworkNodeInterface {
 
   async getMetric(
     rawMetricName: string,
-    comparator: string,
+    comparator?: string,
```

Comment:
```suggestion
    comparator: string,
```

to avoid the non-null assertion

Reply:
This allow us to just get the metric without `comparing` since in this case we want just get the value to operate.
Thanks